docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: '2'
+services:
+  web:
+    image: fpfis/httpd-php-dev:7.4
```

```suggestion
    image: fpfis/httpd-php-dev:8.0
```
We use PHP 8.0 everywhere. Can we use this version here also?